### ISSUE TITLE
Feature Request: Implementing Role-Based Access Control (RBAC) Using multiple Database Integration

### ISSUE THREAD

**Issue by Rachit-Chaudhary11** [zilliztech/attu#402]
**Describe the feature:**
- **Attu Version** 2.3.8
Integrate role management functionality into the Attu user interface to easily manage roles and their assignment to different databases instead of just one database.

**Describe a specific use case for the feature:**
- Create 2 databases - **database1, database2**. Overall now milvus instance will have 3 databases - default, database1 and database2
- Presently you can choose only 1 database from the dropdown above to create a role. Give an option to create roles for multiple databases
- To create a ReadOnly Role for database1, database2. User will need to create separate roles per database

_In short, give an option to create a role for multiple databases.

**Comment by Rachit-Chaudhary11** [comment]
Assign @shanghaikid. Kindly acknowledge the feature request

**Comment by shanghaikid** [comment]
> Assign @shanghaikid. Kindly acknowledge the feature request

I can't promise when, but it's on the roadmap, stay tuned.

**Comment by Rachit-Chaudhary11** [comment]
Thank you @shanghaikid for accepting the feature. Do keep me updated once it is release.

**Comment by shanghaikid** [comment]
please try v2.5.2
